### DISCUSSION TITLE
Clearer Documented "Refresh Rate"?

### DISCUSSION THREAD

**Discussion by klaberte** [breezy-weather/breezy-weather#1013] [Q&A]
I live in the United States, and am using Open-Metro as my data provider.  I am running Breezy Weather on my Pixel (1) Android phone (made by HTC), Running Android 10. My use case is to have this as an always-screen-on appliance, showing the current weather and temperature.  I initially did not allow location services, because I knew that this appliance would not move, and I believed I could simply specify my weather location manually by searching for my closest city.  I also moved the Refresh timing to be 1 hr (60 min).

I also disabled all battery optimizations for Breezy Weather.

What I observed is that this always on screen showing my searched-for location did not auto refresh, nor did I see any issues in the Worker info or when I generated crash logs.  This was disappointing, because I really had hoped to not enable location permissions for this app (not that I have serious concerns, just my general practice not to enable when I do not think they are necessary).

However, if I manually did enable location permissions, and I left the app open to the "Current location" (hiding the location chooser), this screen did update after the refresh time of 60 min.  Still the non-Current, specified-by-search screen still did not update after the refresh time expired.

Perhaps this app requires location services to be on to operate properly.  Perhaps the Auto refresh was only intended for "Current location".  If so, I think a little documentation of this might be helpful.  Perhaps just in in the "Refresh rate" screen, some text like "Only applies to Current location", or something similar.

**Comment by papjul** [comment]
Documentation of background updates is here: https://github.com/breezy-weather/breezy-weather/blob/main/docs/UPDATES.md

If you have issues getting updates in the background to work, have a look at the Troubleshooting section of `Settings` > `Background updates`

**Comment by klaberte** [comment]
Thank you for pointing me to this documentation. I can also report that updating to the latest version 5.2.2 (IIRC) seemed to fix the issue I was describing. I'll close this now.
